course_catalog/tasks_test.py

```diff
@@ -206,6 +206,17 @@ def test_get_ocw_data(settings, mock_course_index_functions):
     assert CoursePrice.objects.count() == 1
     assert CourseInstructor.objects.count() == 1
     assert CourseTopic.objects.count() == 3
+    s3 = boto3.resource(
+        "s3",
+        aws_access_key_id=settings.OCW_LEARNING_COURSE_BUCKET_NAME,
+        aws_secret_access_key=settings.OCW_LEARNING_COURSE_ACCESS_KEY,
+    )
+    # The filename was pulled from the uid 1.json in the TEST_JSON_PATH files.
+    with pytest.raises(botocore.exceptions.ClientError):
+        s3.Object(
+            settings.OCW_LEARNING_COURSE_BUCKET_NAME,
+            "16197636c270e1ab179fbc9a56c72787_master.json",
+        ).load()
```

Comment:
This just seems to be testing that an error is raised when trying to retrieve an object from S3?  I think it would be better if possible to mock `OCWParser.upload_all_media_to_s3` and verify that it is called with argument `upload_master_json=True`

Reply:
We are now properly testing that the file was uploaded.